API/Controllers/CustomerController.cs

```diff
@@ -9,50 +9,49 @@ namespace API.Controllers
     [ApiController]
     public class CustomerController : ControllerBase
     {
-        private readonly ICustomerService _customerService;
+        private readonly ICustomerAppService _appService;
 
-        public CustomerController(ICustomerService customerService)
+        public CustomerController(ICustomerAppService appService)
         {
-            _customerService = customerService ?? throw new ArgumentNullException(nameof(customerService));
+            _appService = appService ?? throw new ArgumentNullException(nameof(appService));
         }
 
         [HttpGet]
-        public IActionResult Get()
+        public virtual IActionResult Get()
         {
             try
             {
-                var result = _customerService.GetAll();
+                var result = _appService.GetAll();
                 return Ok(result);
             }
-            catch (ArgumentException exception)
+            catch
             {
-                var message = exception.InnerException?.Message ?? exception.Message;
-                return BadRequest(message);
+                return NoContent();
             }
         }
 
         [HttpGet("{id}")]
-        public IActionResult GetById(long id)
+        public virtual IActionResult GetById(long id)
         {
             try
             {
-                var result = _customerService.GetById(id);
+                var result = _appService.GetById(id);
                 return Ok(result);
             }
-            catch (ArgumentException exception)
+            catch (ArgumentNullException exception)
             {
                 var message = exception.InnerException?.Message ?? exception.Message;
                 return NotFound(message);
             }
         }
 
         [HttpPost]
-        public IActionResult Create(Customer customer)
+        public virtual IActionResult Create(Customer customer)
         {
```

Comment:
[Comentário Acima](https://github.com/LuanFortunatoW/EverestBackendDotnet/pull/2/files#r1027898541)
```suggestion
        public IActionResult Create(Customer customer)
        {
```